src/ray/protobuf/gcs_service.proto

```diff
@@ -865,4 +866,10 @@ service TaskInfoGcsService {
   rpc GetTaskEvents(GetTaskEventsRequest) returns (GetTaskEventsReply);
 }
 
+// Service for recording one events.
+service OneEventGcsService {
+  // Add OneEvent task data to GCS.
+  rpc AddEvent(events.AddEventRequest) returns (events.AddEventReply);
+}
```

Comment:
"OneEvent" is going to be a very confusing name in the future once the migration is completed. Can we pick something a little more obvious/self describing? Like `EventExportService`